server/models/authmessage.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Data.World Slack Application
```

Comment:
Because you just changed to lowercase everywhere...

Reply:
Ahhh..Good catch! thanks.